scripts/lite/byron-testnet.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+ROOT="$(realpath "$(dirname "$0")/../..")"
+
+configuration="${ROOT}/configuration"
+
+OS=$(uname -s) DATE=
+case $OS in
+  Darwin )       DATE="gdate";;
+  * )            DATE="date";;
+esac
+
+data_dir="$(mktemp).d"
+mkdir -p "${data_dir}"
+
+# Generate genesis keys
+ARGS=(
+  --genesis-output-dir           "${data_dir}/genesis"
+  --start-time                   "$(${DATE} -d "now + 1 minute" +%s)"
+  --protocol-parameters-file     "${ROOT}/scripts/protocol-params.json"
+  --k                            2160
+  --protocol-magic               459045235
+  --n-poor-addresses             128
+  --n-delegate-addresses         7
+  --total-balance                8000000000000000
+  --avvm-entry-count             128
+  --avvm-entry-balance           10000000000000
+  --delegate-share               0.9
+  --real-pbft
+  --secret-seed                  2718281828
+)
+cabal run exe:cardano-cli -- byron genesis genesis "${ARGS[@]}"
```

Comment:
This command was renamed to be clearer and indicate that we are generating a byron genesis